src/core/Table/TableRowMemoized.tsx

```diff
@@ -66,5 +73,7 @@ export const TableRowMemoized = React.memo(
     prevProp.onBottomReached === nextProp.onBottomReached &&
     prevProp.row.original === nextProp.row.original &&
     prevProp.state.selectedRowIds?.[prevProp.row.id] ===
-      nextProp.state.selectedRowIds?.[nextProp.row.id],
+      nextProp.state.selectedRowIds?.[nextProp.row.id] &&
+    prevProp.expandedSubComponent === nextProp.expandedSubComponent &&
+    prevProp.isExpanded === nextProp.isExpanded,
```

Comment:
Wouldnt this be covered with the check of row.original ?

Reply:
Same as my above comment, it wasn't covered by just those prop checks